src/enigma2/data/AutoTimer.cpp

```diff
@@ -158,13 +160,18 @@ bool AutoTimer::UpdateFrom(TiXmlElement* autoTimerNode, Channels &channels)
         m_channelId = channels.GetChannelUniqueId(Channel::NormaliseServiceReference(strTmp.c_str()));
 
         // Skip autotimers for channels we don't know about, such as when the addon only uses one bouquet or an old channel referene that doesn't exist
-        if (m_channelId < 0)
+        if (m_channelId == PVR_CHANNEL_INVALID_UID)
         {
-          Logger::Log(LEVEL_DEBUG, "%s could not find channel so skipping autotimer: '%s'", __FUNCTION__, m_title.c_str());
-          return false;
+          // Logger::Log(LEVEL_DEBUG, "%s could not find channel so skipping autotimer: '%s'", __FUNCTION__, m_title.c_str());
+          // return false;
```

Comment:
I would remove this two commented lines.

Reply:
Done